sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Sample.java

```diff
@@ -58,6 +58,14 @@
     return new SampleAnyCombineFn<>(sampleSize);
   }
 
+  /**
+   * Returns a {@link CombineFn} that computes a single and potentially non-uniform sample value of
+   * its inputs.
+   */
+  public static <T> CombineFn<T, ?, T> anyValueCombineFn() {
```

Comment:
Why `?` as a part of template? (just asking. I am not familiar with how it should be used)

Reply:
To be frank with you, I copied the the method [anyCombineFn](https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Sample.java#L57) and just adapted it. So, I am not sure the reason behind the template usage, I wanted to stick to existing methods.